cmd/libvirt-actuator/utils/clientset_generated.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
This is not related to upstream, so this Copyright notice should probably be removed.